src/renderer.tsx

```diff
@@ -76,8 +76,6 @@ ipcRenderer.on(
     }: { isPackaged: string; locatePastelConfDir: string },
   ) => {
     if (isPackaged) {
-      console.log = log.log
-      console.error = log.error
       log.transports.console.level = false
     }
```

Comment:
Could you please add `else` section with 
```
log.transports.file.level = false
```
As we don't need file log for a not packed app, anyway not bad so I'm not blocking